### PR TITLE
development: Reword domain location to use staging

### PR DESCRIPTION
**Pull request by ZJvandeWeg**
I was confused using staging due to the listed domain. The domain has
not been published to prevent production users of trying the wrong
domain.

### CODE REVIEW

**Review by knolleary — APPROVED** [review]

File: development/staging.md [hunk 1]
```diff
@@ -1,8 +1,8 @@
 # Staging Environment
 
-We have a staging environment running on AWS which is a scaled down replica of FlowForge Cloud.
-
-It uses the domain flowforge.dev
+We have a staging environment running on AWS which is a scaled down replica of
+FlowForge Cloud. Staging URL and sign in details can be found in the
```
Comment (sammachin) [inline development/staging.md:4]:
```suggestion
FlowForge Cloud, with a separate domain. Staging URL and sign in details can be found in the 
```

**Review by sammachin — CHANGES_REQUESTED** [review]
looks fine other than pointing out that it uses a separate domain, even if we don't state what that is

File: development/staging.md (hunk 1 above)
Comment (ZJvandeWeg) [reply development/staging.md:4]: Sure, will push this change to main in a bit

**Comment by knolleary** [comment]
My apologies - I jumped the gun on the merge whilst there were outstanding reviews.

**Pull request merged**